### DISCUSSION TITLE
25 Digit verification code

### DISCUSSION THREAD

**Discussion by anfargaming** [community/community#42767] [GitHub Education]
How do I get a 25 digit verification code for Azure?
Github student developer pack not providing 25 digit unique code
So how i get this code

**Comment by hiten36** [comment]
Visit the azure website by the link in github student developer pack, after that click on start free than sign in with your github account, if this does not help, repeat above step again. You will automatically be redirected to offer page. Or you can simple check for link in the webiste URL. Otherwise just click on Get help at [Microsoft Azure support](https://azure.microsoft.com/en-us/support/options/). Mail them.
